design-proposals/klip-template.md

```diff
@@ -43,13 +43,17 @@ support for data formats? Are we adding / removing configurations? Etc._
 
 ## Design
 
-_How does your solution work?_
+_How does your solution work? This should cover the main data and control flows that are changing._
 
 ## Test plan
 
-_What tests do you plan to write?  What are the failure scenarios that we do / do not cover? It goes 
-without saying that most classes should have unit tests. This section is more focussed on the 
-integration and system tests that you need to write to test the changes you are making._
+_What are the failure scenarios you are going to cover in your testing? What scale testing do you plan to run? What about peformance and load testing? It goes 
+without saying that most classes should have unit tests._
+
+## LOEs and Delivery Milestones
+_Break the work down into more granular tasks. Make sure to include tasks for testing, documentation, etc., in addition to the core development tasks. Each task should be scoped to at most 1 person-week, where a person-week includes buffer time for things like code reviews, design reviews, interviews, etc.._
```

Comment:
this part feels very specific to the team at confluent?  But +1 on having a break down. Can we make the scoping/estimation optional, so that external contributors have less burden..

Reply:
I agree. I updated the text to make it a bit friendlier for people proposing KLIPs externally, but yet retains the requirement to provide breakdowns. Let me know what you think.

Reply:
Break downs are good.. +1